config/examples/sandbox_proxy.yml

```diff
@@ -3,20 +3,14 @@ base: &default
     - echo-api.3scale.net
     - hello-world-api.3scale.net
   ignore_test_failures: []
-  hosts:
-    - localhost:8091
-  backend_host: backend:3000
-  shared_secret: dev
-  backend_scheme: http
-  nginx_port: 8080
+  backend_endpoint: <%= ENV.fetch('APICAST_BACKEND_ROOT_ENDPOINT', 'http://127.0.0.1:3001') %>
```

Comment:
`backend_endpoint` supersedes `backend_host` + `backend_scheme`
https://github.com/3scale/porta/blob/master/app/models/proxy.rb#L324-L325

`shared_secret`, `nginx_port` and ``hosts` don't seem to be used.

Reply:
Also, we need to figure out whether this `APICAST_BACKEND_ROOT_ENDPOINT` is really needed, or we can use `BACKEND_URL` or `BACKEND_PUBLIC_URL`

Reply:
TODO: `sandbox_endpoint` ?